blurb/MANIFEST.in

```diff
@@ -1,2 +0,0 @@
-include LICENSE.txt
-include README.rst
```

Comment:
I added LICENSE.txt and README.rst to the manifest because at one point when I was packaging it complained it didn't have README.rst in the package.  I don't know proper MANIFEST.in hygiene, but--why shouldn't they be listed here?